charts/cromwell/templates/jmxExporterCm.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jmx-exporter-cm
```

Comment:
Prefix this configmap name with `cromwell-` so it doesn't collide with any other Terra apps in same namespace?

Reply:
One more thought -- does it make sense to include this in the regular Cromwell configmap so that changes to it trigger an automatic pod restart?

I think it would just mean changing the Java arg to refer to /etc/cromwell-cm/prometheusJmx.yaml instead

Reply:
good idea. Will make this change